dsc/tests/dsc_sshdconfig.tests.ps1

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'SSHDConfig resource tests' {
+    BeforeAll {
+        $brewExists = ($null -ne (Get-Command brew -CommandType Application -ErrorAction Ignore))
+        $sshdExists = ($null -ne (Get-Command sshd -CommandType Application -ErrorAction Ignore))
+        $isAdmin = if ($IsWindows) {
+            $identity = [System.Security.Principal.WindowsIdentity]::GetCurrent()
+            [System.Security.Principal.WindowsPrincipal]::new($identity).IsInRole([System.Security.Principal.WindowsBuiltInRole]::Administrator)
+        }
+        else {
+            [System.Environment]::UserName -eq 'root'
+        }
+        $runTest = $sshdExists -and $isAdmin
+        $yaml = @'
+$schema: https://aka.ms/dsc/schemas/v3/bundled/config/document.json
+metadata:
+  Microsoft.DSC:
+    securityContext: elevated
+resources:
+- name: sshdconfig
+  type: Microsoft.OpenSSH.SSHD/sshd_config
+  properties:
+'@
+        # set a non-default value in a temporary sshd_config file
+        "LogLevel Debug3" | Set-Content -Path $TestDrive/test_sshd_config
+    }
+
+    AfterAll {
+        if (Test-Path $TestDrive/test_sshd_config) {
+            Remove-Item -Path $TestDrive/test_sshd_config -Force
+        }
+    }
+
+    It 'Export works' {
+        if ($runTest) {
+            $out = dsc config export -i "$yaml" | ConvertFrom-Json -Depth 10
+            $LASTEXITCODE | Should -Be 0
+            $out.resources.count | Should -Be 1
+            $out.resources[0].properties | Should -Not -BeNullOrEmpty
+            $out.resources[0].properties.port[0] | Should -Be 22
+        }
+    }
+
+    It 'Get works'{
```

Comment:
Is there any real difference in this case between `get` and `export`?  If not, just have it as one test with `-TestCases` to perform the different operations but have the same validation code